test/test_BE.js

```diff
@@ -0,0 +1,20 @@
+const tape = require('tape');
+const supertest = require('supertest');
+const app = require('./../src/app');
+
+
+// Test Doctors Route
+tape('check doctor.get route ', (t) => {
```

Comment:
the description about doctor rout but the test for /chat route!!

change the description and add a test for /doctors route